pychunkedgraph/io/protobuf/chunkEdges.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package test;
+
+message ChunkEdges {
+  message Edges {
```

Comment:
Would like to check how badly the decoding performance suffers if we change this to the actual values we expect (something like `repeated fixed64`) - that makes it clearer on what the message actually contains and saves the double decoding in [_decompress_edges](https://github.com/seung-lab/PyChunkedGraph/pull/153/files#diff-6d5b90ec37c8ad44b65000ef14e6c421R34-R36)